sc-memory/sc-core/sc-store/sc_iterator3.c

```diff
@@ -593,14 +592,14 @@ sc_bool _sc_iterator3_a_a_f_next(sc_iterator3 * it)
 
 sc_bool _sc_iterator3_a_f_a_next(sc_iterator3 * it)
 {
-  if (!it->finished)
+  if (it->finished == SC_FALSE)
```

Comment:
I consider it is normal to write conditions in this form and in this form they were mostly written here, since the exclamation mark cannot always be seen.